.flowconfig

```diff
@@ -13,6 +13,6 @@
 
 [options]
 module.system.node.resolve_dirname=node_modules
-module.name_mapper.extension='scss' -> 'empty/object'
+module.name_mapper='.*\(.s?css\)' -> 'empty/object'
```

Comment:
This changes are also done in https://github.com/3scale/porta/pull/1594
You may need to rebase